var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -33,8 +33,7 @@ class Nwchem(Package):
     homepage = "http://www.nwchem-sw.org"
     url      = "http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz"
 
-    version('6.6', 'c581001c004ea5e5dfacb783385825e3',
-            url='http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz')
+    version('6.6.revision27746-src.2015-10-20', 'c581001c004ea5e5dfacb783385825e3')
```

Comment:
I'm not sure about this one. I can't tell if the revision is a more specific version, or if it is extraneous information. Can we reasonably guarantee that there will not be another revision of 6.6 released?